app/views/layouts/application.html.erb

```diff
@@ -5,8 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
     <title><%= content_for?(:title) ? yield(:title) : "foundation-rails" %></title>
+    <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600|Ubuntu:400,500' rel='stylesheet' type='text/css'>
 
-    <%= stylesheet_link_tag    "application" %>
+    <%= stylesheet_link_tag    "application", debug: false %>
```

Comment:
the `debug:false` allowed me to see what it would look like with the stylesheets all concatenated together into one file. This can be removed.